workitem/undoable_wi_repo.go

```diff
@@ -33,12 +33,12 @@ type UndoableWorkItemRepository struct {
 }
 
 // Load implements application.WorkItemRepository
-func (r *UndoableWorkItemRepository) Load(ctx context.Context, ID string) (*app.WorkItem, error) {
```

Comment:
Same with this. Patched, but not used. I think on a technical level what it does is replacable with the much simpler DeleteEntries Gorm plugin.